packages/wrangler/src/__tests__/kv.test.ts

```diff
@@ -528,7 +528,7 @@ describe("wrangler", () => {
 					},
 				});
 				await runWrangler(
-					`kv:key put dKey dVal --namespace-id some-namespace-id --metadata {"mKey":"mValue"}`
+					`kv:key put dKey dVal --namespace-id some-namespace-id --metadata '{"mKey":"mValue"}'`
```

Comment:
This wasn't a correct example of how a user would specify the metadata arg with JSON.

Before (incorrect, note the missing quotes which would fail JSON.parse)
```sh
▶ node -e 'console.log(process.argv.slice(1))' --  --metadata {"mKey":"mValue"}  
[ '--metadata', '{mKey:mValue}' ]
```

After (correct, note the valid JSON as expected):
```sh
▶ node -e 'console.log(process.argv.slice(1))' --  --metadata '{"mKey":"mValue"}'
[ '--metadata', '{"mKey":"mValue"}' ]
```